src/uwtools/tests/fixtures/hello_workflow.yaml

```diff
@@ -0,0 +1,38 @@
+rocoto:
+  attrs:
+    realtime: false
+    scheduler: slurm
+  cycledefs:
+    howdy: 
+      - 202209290000 202209300000 06:00:00
+  entities:
+    ACCOUNT: myaccount
+    FOO: test.log
+  log: /some/path/to/&FOO;
+  tasks:
+    task_hello:
+      attrs:
+        cycledefs: howdy
+        maxtries: "2"
+      account: "&ACCOUNT;"
+      command: "echo hello $person"
+      jobname: hello
```

Comment:
Taking a look at this again, is `jobname` suppose to be added here in the yaml? I believe we are adding jobname using the `_add_jobname` function and the yaml shouldn't be providing this? If that's the case then I'm not sure `_add_jobname `is correctly iterating through the config_obj passed from write_xml_rocoto. If jobname is suppose to be included here then should we be including it below for `task_hello_#member#:` as well? @christinaholtNOAA

Reply:
This entry likely should not go here.